Ignore links in comments *TEMP BANDAID*

Acidion opened the pull request:
Added 2 regular expressions to remove all HTML tags from the comment
(line 139) and to remove any links that might still remain due to being
the text of the hyperlink (line 141).

These changes should keep the bot from responding to 'AFSCs' that just
happen to be part of a URL as opposed to someone actually referencing an
AFSC.

Acidion committed:
Ignore links in comments *TEMP BANDAID*

Added 2 regular expressions to remove all HTML tags from the comment
(line 139) and to remove any links that might still remain due to being
the text of the hyperlink (line 141).

These changes should keep the bot from responding to 'AFSCs' that just
happen to be part of a URL as opposed to someone actually referencing an
AFSC.

HadManySons commented:
Doesn't work. Tested the regex on regex101.com, it all matches fine. Appears to be an issue with the re.sub() function. Also, please convert all tabs to spaces when committing. I'm guilty of it too, but I'm try to keep the repo as PEP-8 compliant as possible.

HadManySons committed:
updated to return to formattedComment object

with the update, the re.sub() call returned to nothing.

Acidion committed:
String Literal Addition to Regex

Looks like the https regex needed the raw string literal character on it (r'). No clue if this will show up with tab or spaces, just using the online editor on it. Forgive me, PEP-8.

Tested on two comments that broke the bot with the online compiler/interpreter here: https://repl.it/JJTo/1

HadManySons commented:
A major overhaul of the bot is imminent, so this pull request won't be needed anymore. Thanks for all the hard work though.

HadManySons commented:
Re opened because there's still value in this pull request to search for AFSCs that are NOT already included in a link somewhere. After the major update is committed into the main repo, let's get this pull request going to exclude things that already linked or included as part of a link.

HadManySons commented:
Major bot update. This pull request can still fix an issue if we can figure out why it wasn't working originally and update it for the new code format.
